src/engine/source/server/src/endpoints/unixStream.cpp

```diff
@@ -0,0 +1,354 @@
+#include <server/endpoints/unixStream.hpp>
+
+#include <sys/un.h> // Unix socket datagram bind
+
+#include <logging/logging.hpp>
+
+namespace engineserver::endpoint
+{
+
+UnixStream::UnixStream(const std::string& address,
+                       std::shared_ptr<ProtocolHandlerFactory> factory,
+                       const std::size_t taskQueueSize,
+                       std::size_t timeout)
+    : Endpoint(address, taskQueueSize)
+    , m_handle(nullptr)
+    , m_timeout(timeout)
+    , m_factory(std::move(factory))
+{
+    if (m_timeout == 0)
```

Comment:
Please invert the condition, same for every ocurrence

Reply:
done